perma_web/perma/settings/deployments/settings_common.py

```diff
@@ -157,7 +157,6 @@ def _pywb_static_dir_location():
 
```

Comment:
I think it would be prudent to get rid of it, delete the variable from settings on all the servers, and get rid of it in this template:

~perma_upstream/perma_web/perma/templates/js_config.html:
```
    1  <script>
    2    var settings = {
    3:     API_VERSION: {{ API_VERSION }},
    4      STATIC_URL: "{{ STATIC_URL }}",
    5      MEDIA_URL: "{{ MEDIA_URL }}",
    .
    7      DEBUG: {{ DEBUG|lower }}
    8    },
    9:   api_path = "/api/v" + settings.API_VERSION
   10  </script>
```
What do you think?